release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -521,6 +521,11 @@ In the following tables, features are marked with the following statuses:
 |Removed
 |Removed
 
+|Installing a cluster on {rh-openstack-first} with Kuryr
+|Deprecated
+|Deprecated
+|Removed
+
```

Comment:
Kuryr item already exists in lines 591: https://github.com/openshift/openshift-docs/pull/70051/files#diff-a4394e49205b55005aa75bf885c9f0d88e144344ff0c87ebe59d856269313337L586-L596

Reply:
Yep, it sure does 😆 (doh). Thank you.